.env_example

```diff
@@ -0,0 +1,16 @@
+; Oauth parameters.
+OAUTH_CONSUMER_TOKEN =
+OAUTH_SECRET_TOKEN   =
+OAUTH_ENDPOINT       = 'https://meta.wikimedia.org/w/index.php?title=Special:OAuth'
+OAUTH_REDIR          = 'https://meta.wikimedia.org/wiki/Special:OAuth/authenticate?'
+OAUTH_CALLBACK       = 'oob'
+
+; Database connection details. The same username and password is used for both data sources.
+DB_DSN_ENWIKI      = "mysql:host=127.0.0.1;port=4711;dbname=enwiki_p"
+DB_DSN_PLAGIABOT   = "mysql:host=127.0.0.1;port=4711;dbname=s51306__copyright_p"
+DB_USER            =
+DB_PASS            =
+
+; Caching system. If a Redis host is not defined, a local Filesystem cache will be used instead.
+REDIS_HOST = "tools-redis"
+REDIS_PORT = 6379
```

Comment:
We might want to add `SLIM_MODE = 'development'` here, or at least I have on my local. Looking at the core Slim code I'm not convinced this doesn't do anything in particular by itself, but we might (or should) use it in other parts of the app, e.g. https://github.com/wikimedia/CopyPatrol/blob/master/src/Controllers/CopyPatrol.php#L364